fetch.bs

```diff
@@ -6706,30 +6671,26 @@ constructor, when invoked, must run these steps:
 
    <li><p>Let <var>Content-Type</var> be null.
 
-   <li><p>Set <var>r</var>'s <a for=Response>response</a>'s <a for=response>body</a> and
+   <li><p>Set <a>this</a>'s <a for=Response>response</a>'s <a for=response>body</a> and
    <var>Content-Type</var> to the result of <a for=BodyInit>extracting</a> <var>body</var>.
 
-   <li><p>If <var>Content-Type</var> is non-null and <var>r</var>'s <a for=Response>response</a>'s
+   <li><p>If <var>Content-Type</var> is non-null and <a>this</a>'s <a for=Response>response</a>'s
    <a for=response>header list</a> <a for="header list">does not contain</a>
    `<code>Content-Type</code>`, then <a for="header list">append</a>
-   `<code>Content-Type</code>`/<var>Content-Type</var> to <var>r</var>'s
+   `<code>Content-Type</code>`/<var>Content-Type</var> to <a>this</a>'s
    <a for=Response>response</a>'s <a for=response>header list</a>.
   </ol>
 
- <li><p>Set <var>r</var>'s <a for=Body>MIME type</a> to the result of
- <a for="header list">extracting a MIME type</a> from <var>r</var>'s <a for=Response>response</a>'s
+ <li><p>Set <a>this</a>'s <a for=Body>MIME type</a> to the result of
+ <a for="header list">extracting a MIME type</a> from <a>this</a>'s <a for=Response>response</a>'s
  <a for=response>header list</a>.
 
- <li><p>Set <var>r</var>'s <a for=Response>response</a>'s
- <a for=response>HTTPS state</a> to
- <a>current settings object</a>'s
+ <li><p>Set <a>this</a>'s <a for=Response>response</a>'s <a for=response>HTTPS state</a> to
+ <a>this</a>'s <a>relevant settings object</a>'s
  <a for="environment settings object">HTTPS state</a>.
-
- <li><p>Return <var>r</var>.
 </ol>
 
-<p>The static <dfn method for=Response><code>error()</code></dfn> method, when invoked, must run
-these steps:
+<p>The static <dfn method for=Response><code>error()</code></dfn> method steps are:
 
 <ol>
  <li><p>Let <var>r</var> be a new {{Response}} object, whose <a for=Response>response</a> is a new
```

Comment:
This could link to https://heycam.github.io/webidl/#new and state the realm (current realm).

There are a total of 6 nearby object creations that could do the same.